SU-HSDO/suhumsci
# HSD8-682 Fix table pattern when used in views

# READY FOR REVIEW

# Summary
- Fix table pattern when using it in views.

# Needed By (Date)
- 7/31

# Urgency
- medium

# Steps to Test
1. checkout this branch.
1. temporarily edit `docroot/sites/blt.development.services.yml` file and set `debug: false` (this keeps html comments and stuff from showing up in the dom, so you can simulate prod site)
1. `blt drupal:sync --site=mathematics --partial`
1. go to `/test-table-view` and ensure each row has the same number of columns.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
